app/uk/gov/hmrc/alcoholdutyreturns/service/AccountService.scala

```diff
@@ -35,13 +35,13 @@ class AccountServiceImpl @Inject() (
 
   def createUserAnswers(
     userAnswers: UserAnswers
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): EitherT[Future, ErrorResponse, UserAnswers]         =
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): EitherT[Future, ErrorResponse, UserAnswers] =
     for {
       subscription <- checkSubscriptionStatus(userAnswers.id.appaId)
```

Comment:
Another one for a later refactor, we have the regimes and obligation needed for the audit, but we then have to get them back out of the json (and the boilerplate to handle it going wrong), it would be better for these to be fetched via a call at the top level (controller or service) and then added to the user answers so they can be reused when it comes to auditing.